doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+is a library for writing Plutus contracts. It provides the Contract Monad interface 
+for writing the off-chain part of a Plutus application that is to be interpreted 
+by an emulator or a plutus application backend (PAB). 
+
++--------------+---------------------------------------------------------------+
+| Contract monad                                                               |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core            |
++--------------+---------------------------------------------------------------+
+| State        | Maintained but not extended. Unstable interface.              | 
++--------------+---------------------------------------------------------------+
+
+8. Contract monad emulator
+-----------------------------------------------------------
+
+`Contract monad emulator <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides an environment for emulating the blockchain. The environment 
+provides a way for writing traces for the contract which are sequences of actions 
+by simulated wallets that use the contract. The component is highly dependent on the 
+Contract Monad. However, there are plans to decouple it from the Contract Monad 
+so that it can be used with other libraries.
+
++--------------+---------------------------------------------------------------+
+| Contract monad emulator                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+| Dependencies | cardano-api, plutus-ledger-api, plutus-ledger,                |
+|              | plutus-ledger-constraints, plutus-chain-index-core,           |
+|              | plutus-contract                                               |
++--------------+---------------------------------------------------------------+
+| State        | Maintained and will be extended. Unstable interface.          | 
++--------------+---------------------------------------------------------------+
+
+9. Plutus contract state machine
+-----------------------------------------------------------
+
+`Plutus contract state machine <https://github.com/input-output-hk/plutus-apps/tree/main/plutus-contract>`_ 
+is a library that provides a high-level model for writing a Plutus application based 
+on the State Machine formalism. Even though it is inefficient, it can be used to 
+model a Plutus application and to test the actual implementation.
+
```

Comment:
Generally this shouldn't be used in production as the scripts are too bit to run on chain.

Reply:
Noted.